lib/src/file_util.rs

```diff
@@ -29,6 +35,14 @@ pub struct PathError {
     pub error: io::Error,
 }
 
+#[derive(Debug, Error)]
+pub enum SymlinkError {
+    #[error("Developer Mode not enabled")]
+    DeveloperModeNotEnabled,
+    #[error("Symlink creation failed")]
+    InternalError(#[source] std::io::Error),
```

Comment:
nit: `Io(..)` instead of `InternalError(..)`?

Reply:
Fixed